opencti-platform/opencti-front/src/components/dataGrid/components/DataTableBody.tsx

```diff
@@ -204,71 +79,54 @@ const DataTableBody = ({
     return onToggleEntity(currentEntity, event);
   };
 
+  const [tableHeight, setTableHeight] = useState(0);
   useLayoutEffect(() => {
-    const handleResize = () => setResize(true);
-    const handleStorage = ({ key }: StorageEvent) => setTimeout(() => {
-      if (key === 'navOpen') {
-        setResize(true);
-      }
-    }, 200);
+    const hasFilters = (filters?.filters ?? []).length > 0;
+    let filtersHeight = hasFilterComponent ? 54 : 0;
+    if (hasFilterComponent && hasFilters) filtersHeight += 48;
 
-    window.addEventListener('resize', handleResize);
-    window.addEventListener('storage', handleStorage);
-    if (rootRef) resizeObserver.current.observe(rootRef);
-    let observer: MutationObserver | undefined;
-    const elementToObserve = document.getElementById('filter-container');
-    if (elementToObserve) {
-      observer = new MutationObserver(() => setResize(true));
-      observer.observe(elementToObserve, { childList: true });
+    if (variant === DataTableVariant.widget) {
+      if (!rootRef) throw Error('Invalid configuration for widget list');
+      setTableHeight(rootRef.offsetHeight + 50);
+    } else if (rootRef) {
+      setTableHeight(rootRef.offsetHeight - filtersHeight);
+    } else {
+      // TODO: this computation should be avoided because too many risk of changes.
```

Comment:
is it a forgotten TODO?

Reply:
Nop, just to not forget but not the scope of this PR because it means revamp majority of usages